src/routes/[deviceUID]/dashboard/+page.svelte

```diff
@@ -279,9 +284,19 @@
         </a>
       </p>
 
-      <div class="map">
-        <MapboxMap {lastReading} />
-      </div>
+      {#if hideMap === 0}
+        <div class="map">
+          <MapboxMap {lastReading} />
+        </div>
+      {:else}
+        <div class="banner">
+          <p>
+            Not seeing your device on the map? Check the 
+            <a href="/{deviceUID}/DeviceSettings">settings page</a> 
```

Comment:
this href is wrong, this is the URL pattern it should follow `href="/{deviceUID}?product={productUID}&pin={pin}&internalNav=true`